docs/basics/networkobject.md

```diff
@@ -123,7 +123,7 @@ To find your own player object just pass `NetworkManager.Singleton.LocalClientId
 
 ## Network Prefabs
 
-Network Prefabs (NetworkPrefabs) are Prefabs that contain a GameObject with a NetworkObject component. As an example, if you wanted to create a Prefab to be the default player Prefab, then you would create a Prefab that at the root GameObject included a NetworkObject component and any additional player specific NetworkBehaviour components. You can then assign that Prefab to the `NetworkManager` Player Prefab property to use when a player connects and is approved. Each connected player has a unique instance spawned on all connected clients (including the server).
+Network Prefabs (NetworkPrefabs) are Prefabs that contain a GameObject with a NetworkObject component. As an example, if you wanted to create a Prefab to be the default player Prefab, then you would create a Prefab that at the root GameObject included a NetworkObject component and any additional player specific NetworkBehaviour components. You can then assign that Prefab to a `ScriptableObject` Player Prefab property to use when a player connects and is approved. Each connected player has a unique instance spawned on all connected clients (including the server).
```

Comment:
This isn't quite accurate.

The accurate thing to say is:

> Network Prefabs are registered to a `NetworkPrefabsList` object (a type of `ScriptableObject`). By default, a Default Prefabs List is created that contains every Network Prefab in your project. However, for cases where you want to limit which prefabs are available (for example, to reduce memory usage), you can disable this behavior in `Project Settings > Netcode For GameObjects > Project Settings`, and a `NetworkPrefabsList` can be manually created by right-clicking in the assets view and selecting `Create > Netcode > Network Prefabs List` and adding your prefabs to it. That prefab list can then be assigned to a `NetworkManager` to allow that `NetworkManager` to create those prefabs.

Feel free to massage the wording there.

Reply:
Ah, that makes sense